pinger.go

```diff
@@ -29,33 +29,69 @@ type Pinger struct {
 	requests map[uint16]*request // currently running requests
 	mtx      sync.RWMutex        // lock for the requests map
 	id       uint16
-	conn     *icmp.PacketConn
+	conn4    *icmp.PacketConn
+	conn6    *icmp.PacketConn
 	wg       sync.WaitGroup
 }
 
 // New creates a new Pinger. This will open the raw socket and start the
 // receiving logic. You'll need to call Close() to cleanup.
-func New(bind string) (*Pinger, error) {
-	// Socket öffnen
-	conn, err := icmp.ListenPacket("ip4:icmp", bind)
+func New(bind4, bind6 string) (*Pinger, error) {
+	// open sockets
+	conn4, err := connectICMP("ip4:icmp", bind4)
 	if err != nil {
 		return nil, err
 	}
 
+	conn6, err := connectICMP("ip6:ipv6-icmp", bind6)
```

Comment:
Don't bind if `bind6 == ""`

Reply:
This is handled in `connectICMP` for both `bind4` and `bind6`.